modules/ROOT/pages/provisioning-gcp.adoc

```diff
@@ -85,3 +85,40 @@ gcloud compute instances create                \
 
 NOTE: By design, https://cloud.google.com/compute/docs/startupscript[startup scripts] are not supported on FCOS. Instead, it is recommended to encode any startup logic as systemd service units in the Ignition configuration.
 Again, note you need to use the `user-data` key for Ignition; it will also not work to paste Ignition into this field in the web console.
+
+
+== Launch a Confidential VM
+
```

Comment:
Can you add the same warning that we added for Azure here and we'll merge this one as well? Thanks